cmd/exo/cmd/vm_delete.go

```diff
@@ -24,46 +24,54 @@ var vmDeleteCmd = &cobra.Command{
 			return err
 		}
 
+		tasks := []task{}
+
 		for _, arg := range args {
-			if err := deleteVM(arg, force); err != nil {
-				_, err = fmt.Fprintf(os.Stderr, err.Error())
-				if err != nil {
+			tsk, err := prepareDeleteVM(&arg, force)
```

Comment:
why a pointer?

Reply:
If you read `prepareDeleteVM(&arg, force)`function code I need pointer to update `arg` to be sure it's a name and not an ID because i display it in L35 `fmt.Sprintf("Destroying %q ", arg)`

Reply:
it's butt ugly. I'd move the retrieval of the VM here.

Reply:
yes agree ! I think it was slackness